scripts/check-and-update-submodule.sh

```diff
@@ -46,26 +46,36 @@ if [ "$#" -ne 1 ]; then
     exit 1
 fi
 
+echo "HI4"
```

Comment:
since this file isnt being used by any runner (since the test is [commented out](https://github.com/Volumetrics-io/mrjs/pull/607/files#r1588509898), leaving this in as i debug this over time, no harm no foul